css/properties/align-items.json

```diff
@@ -411,6 +411,57 @@
               "standard_track": true,
               "deprecated": false
             }
+          },
+          "start_end": {
```

Comment:
This duplicates the `"start_end"` feature above; the data can go there for the most part (leave the Firefox 45 data intact, for instance)

Reply:
The `start_end` <i>sub-sub-</i>features actually belong to two different <i>sub-</i>features:
`flex_context` and `grid_context`. So they shouldn't be duplicates 😀